include/deal.II/base/vectorization.h

```diff
@@ -221,7 +223,7 @@ class VectorizedArray
    */
   DEAL_II_ALWAYS_INLINE
   VectorizedArray &
-  operator+=(const VectorizedArray<Number> &vec)
+  operator+=(const VectorizedArray<Number, width> &vec)
```

Comment:
For consistency with the specialized classes further down, can you please write this as follows:
```suggestion
  operator+=(const VectorizedArray &vec)
```
here and for the following functions? I know this is pre-existing, but it's good to be consistent.